PublicMapping/districtbuilder
# Add keyboard shortcuts for map actions

## Overview

Adds a number of keyboard shortcuts to allow power users to toggle map features without clicking UI elements.

#### Implemented in this PR as written in #266 

Key | Action
-- | --
E | Previous district
D | Next district
S | Use bigger geolevels
F | Use smaller geolevels
W | Previous selection tool
R | Next selection tool
G | Accept changes
A | Reject changes
1 | Toggle population labels
Q | Toggle lock on selected district
⌘Z | Undo
SHIFT⌘Z | Redo
C | Limit draw to containing geounit

#### Suggested additional action included here
--|--
P|Toggle evaluate mode


#### Previously implemented in PR #687 
--|--
SPACE | Hold to pan map when using rectangle or brush

#### Not implemented here
--|--
X | Limit draw to unassigned

### Checklist


### Demo

Optional. Screenshots, examples, etc.

### Notes

After adding the core infrastructure for handling keyboard shortcuts in #687 , it's actually pretty easy to handle these map actions here given that all except one of these functions already exist as redux actions. I migrated `setMapLabel` from existing as a `useState` on `ProjectScreen` to a new redux action to make it easier to access.

## Testing Instructions

- Open a map
- Test all of the toggles listed in the table above to confirm that they do what they are indicated to do
- Expect toast message to be displayed for all changes that aren't _visually obvious_ (IMO there is only one of these... but let me know if you feel that it is needed for other actions)

Closes #266
